server/lib/job-queue/job-queue.ts

```diff
@@ -154,6 +160,15 @@ class JobQueue {
       logger.error('Unknown queue %s: cannot create job.', obj.type)
       return
     }
+    if (obj.type === 'video-transcoding') {
```

Comment:
Prefer to create a `createTranscodingJob` wrapper in `lib/videos` that increment the counter